resources/testing/statements/DWH/calculations.sql

```diff
@@ -7,69 +10,60 @@ DELETE FROM public.calculations;
 -- Insert organisations, important to note for non production environments, the email address must one setup in our GOV.UK Notify account
 INSERT INTO public.organisations (sbi, "addressLine1", "addressLine2", "addressLine3", city, county, postcode, "emailAddress", frn, name, updated)
 VALUES
-  (105000001, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000001, 'Mr A Farmer', NOW()),
-  (105000002, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000002, 'Mr A Farmer', NOW()),
-  (105000003, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000003, 'Mr A Farmer', NOW()),
-  (105000004, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000004, 'Mr A Farmer', NOW()),
-  (105000005, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000005, 'Mr A Farmer', NOW()),
-  (105000006, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000006, 'Mr A Farmer', NOW()),
-  (105000007, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000007, 'Mr A Farmer', NOW()),
-  (105000008, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000008, 'Mr A Farmer', NOW()),
-  (105000009, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000009, 'Mr A Farmer', NOW()),
-  (105000010, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000010, 'Mr A Farmer', NOW()),
-  (105000011, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000011, 'Mr A Farmer', NOW()),
-  (105000012, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000012, 'Mr A Farmer', NOW()),
-  (105000013, 'A Farm', 'A Place', 'A Land', 'A City', 'A County', 'NE1 2FA', 'john.watson2@rpa.gov.uk', 1000000013, 'Mr A Farmer', NOW());
+(110100001,'A farm','A field','A place','A town','A county','FA1 2PY','john.watson2@rpa.gov.uk',1000000001,'Mr Unsettled Top Up',NOW()),
```

Comment:
kindly update this `john.watson2@rpa.gov.uk` to `<enter your email here>` so each script is customised to whoever is running it locally.

Thoughts?